composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
+        "php": ">=5.5",
```

Comment:
Not a personal choice, as noted in the commit message, the zend libs require this for me. You don't have that issue?